tempest/api/workloadmgr/restore/test_exclude_cinder_volume.py

```diff
@@ -0,0 +1,449 @@
+                raise Exception(
+                    "Command workload create did not execute correctly")
+            else:
+                reporting.add_test_step(
+                    "Execute workload-create command using cli",
+                    tvaultconf.PASS)
+
+            time.sleep(10)
+
+            # check workload status
+            wid = query_data.get_workload_id_in_creation(tvaultconf.workload_name)
+            LOG.debug("Workload ID: " + str(wid))
+            if (wid is not None):
+                self.wait_for_workload_tobe_available(wid)
+                if (self.getWorkloadStatus(wid) == "available"):
+                    reporting.add_test_step(
+                        "Create workload using cli", tvaultconf.PASS)
+                else:
+                    reporting.add_test_step(
+                        "Create workload using cli", tvaultconf.FAIL)
+                    raise Exception("Create workload using cli failed.")
+            else:
+                reporting.add_test_step(
+                    "Create workload using cli", tvaultconf.FAIL)
+                raise Exception("Create workload using cli failed.")
+
+            # take full snapshot
+            snapshot_id = self.workload_snapshot(wid, True)
+
+            self.wait_for_workload_tobe_available(wid)
+            snapshot_status = self.getSnapshotStatus(wid, snapshot_id)
+
+            if (snapshot_status == "available"):
+                LOG.debug("Full snapshot created.")
+                reporting.add_test_step("Create full snapshot", tvaultconf.PASS)
+            else:
+                reporting.add_test_step("Create full snapshot", tvaultconf.FAIL)
+                raise Exception("Full snapshot creation failed.")
+
+            # delete few files from original vm instance
+            self.data_ops_delete_files(floating_ip_1, mount_point[0], 1)
+            time.sleep(10)
+
+            ### Selective restore ###
+            rest_details = {}
+            rest_details['rest_type'] = 'selective'
+            rest_details['network_id'] = CONF.network.internal_network_id
+            rest_details['subnet_id'] = self.get_subnet_id(
+                CONF.network.internal_network_id)
+            volumeslist = []
+            rest_details['instances'] = {vm_id: volumeslist}
+
+            payload = self.create_restore_json(rest_details)
+            # Trigger selective restore
+            restore_id_1 = self.snapshot_selective_restore(
+                wid,
+                snapshot_id,
+                restore_name=tvaultconf.restore_name,
+                restore_cleanup=True,
+                instance_details=payload['instance_details'],
+                network_details=payload['network_details'])
+
+            self.wait_for_snapshot_tobe_available(wid, snapshot_id)
+            if (self.getRestoreStatus(wid, snapshot_id, restore_id_1) == "available"):
+                reporting.add_test_step("Selective restore of snapshot id = {} ".format(
```

Comment:
Including snapshot id in report will not help , instead u need to include type of snapshot. Like selective restore of full snapshot. That will help team identify which snapshot is restored.